example/server.js

```diff
@@ -2,22 +2,44 @@
 
 var Hapi = require('hapi');
 var Hoek = require('hoek');
-var tags = require('../lib/index.js');
+var fs = require('fs');
 
 var server = new Hapi.Server();
+var tagsData = JSON.parse(fs.readFileSync('./example/tags.json').toString());
+var tags = require('../lib/index.js');
+var tagsPool = require('./pg.js');
+
+function init (port, pgConfig, callback) {
+  server.connection({ port: port });
+
+  server.register([{
+    register: tags,
+    options: { tags: tagsData, tagsPool: tagsPool(pgConfig) }
```

Comment:
yes I was thinking someting like this :+1: but still I don't this we need the tagsPool function at the moment

```
function init (port, pgConfig, callback) {
   server.connection({ port: port });
   var pool =  new pg.Pool(config);
  server.register([{
     register: tags,
     options: { tags: tagsData, tagsPool: pool }
```